modules/ROOT/pages/trusted-salesforce-org.adoc

```diff
@@ -1,86 +1,33 @@
-* A connection code from your Salesforce organization
-
-== Create a Connection
-
-To initiate the connection creation process, you need the connection code from your Salesforce organization.
-
-include::partial$include-nav-steps-org-admin.adoc[]
-
-. Click *External Access*.
-. In the *External Access* page, click *Salesforce*.
-. Click *Connect with a new Salesforce Org*.
-. Enter your connection code from your Salesforce organization.
-. Click *Confirm*. +
-After the connection completes on the Anypoint Platform side, a connection code for your Anypoint Platform organization appears.
-+
-. Copy the Anypoint Platform connection code.
-. Return to the Salesforce connection creation interface, and enter your Anypoint Platform connection code.
-. Complete the instructions in Salesforce.
-
-The organization relationship appears in the *Salesforce* section of *External Access* with the `Enabled` status.
-
-== Disconnect or Reconnect an Existing Connection
-
-If you want to disconnect your Salesforce organization from your Anypoint Platform organization, you can use the Salesforce user interface to do so. If you want to resume publishing assets from Anypoint Platform to Salesforce after you disconnect the Salesforce organization, you must reconnect the organizations. 
-
-Alternatively, if you want to temporarily prevent Anypoint Platform users from publishing assets to Salesforce from the Anypoint Platform side, you can instead disable the connection temporarily.
-
-For more information about managing this relationship from the Salesforce user interface, see the https://help.salesforce.com/s/articleView?id=sf.external_services_manage_your_mulesoft_anypoint_platform_connection.htm&type=5[Salesforce documentation]. 
-
-=== Disconnect Your Organizations
-
-. In the Salesforce user interface, select your Anypoint Platform organization.
-. Click the overflow menu icon.
-. Click *Disconnect*.
-
-=== Reconnect Your Organizations
-
-. In the Salesforce user interface, select your Anypoint Platform organization.
-. Click the overflow menu icon.
-. Click *Reconnect*.
-
-== Enable or Disable a Connection to Your Organization
+== Salesforce Organization Administrators
 
-When you enable the connection with your Salesforce organization, you can publish assets from your Anypoint Platform organization to the trusted Salesforce organization. When you disable the connection, the Salesforce organization still appears in the list of available organizations but you can no longer publish assets from your Anypoint Platform to your Salesforce organization. Disabling a connection does not sever the connection to your Salesforce organization. 
+When connecting an Anypoint Platform organization with a Salesforce org, a Salesforce administrator can: 
 
-include::partial$include-nav-steps-org-admin.adoc[]
+* Complete the connection process to the Anypoint Platform organization.
+* Disconnect or reconnect an Anypoint Platform and Salesforce org.
```

Comment:
"Anypoint Platform org"?

Reply:
We don't use "org" in AM documentation anywhere. It's better to use "organization" everywhere to stay consistent with the rest of AM documentation.